app/src/main/res/layout-land/option_app_language.xml

```diff
@@ -13,15 +13,19 @@
     android:id="@+id/app_language_item_layout"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:background="@drawable/general_item_background_border"
+    android:minHeight="48dp"
     android:onClick="@{(v) -> viewModel.onAppLanguageClicked()}"
-    android:background="@color/white">
+    android:paddingStart="36dp"
+    android:paddingTop="20dp"
+    android:paddingEnd="36dp"
+    android:paddingBottom="20dp">
 
     <TextView
       android:id="@+id/app_language_label_text_view"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:fontFamily="sans-serif"
-      android:padding="8dp"
       android:text="@string/app_language"
       android:textColor="@color/oppiaPrimaryText"
```

Comment:
In the mock textcolor is #000000DE.

Reply:
Done in 3 landscape and 3 portrait files.